recipes/d3ploy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "d3ploy" %}
+{% set version = "0.8" %}
+{% set sha256 = "f98c4984974a834963b474be394f10aac718d72ba5ea48c0da2ca098ff202943" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ergs/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Shouldn't this be `url: https://github.com/ergs/{{ name }}/archive/{{ version }}.tar.gz`